src/app-check/app-check-api.ts

```diff
@@ -0,0 +1,98 @@
+/*!
+ * @license
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Interface representing an App Check token.
+ */
+export interface AppCheckToken {
+  /**
+   * The Firebase App Check token.
+   */
+  token: string;
+
+  /**
+   * The time-to-live duration of the token in milliseconds.
+   */
+  ttlMillis: number;
+}
+
+/**
+ * Interface representing a decoded Firebase App Check token, returned from the
+ * {@link appCheck.AppCheck.verifyToken `verifyToken()`} method.
```

Comment:
This needs to follow the conventions described at https://tsdoc.org/pages/tags/link/

Reply:
Thanks! Fixed the links.